app/src/main/java/nerd/tuxmobil/fahrplan/congress/sharing/SessionSharer.kt

```diff
@@ -0,0 +1,36 @@
+package nerd.tuxmobil.fahrplan.congress.sharing
+
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.content.Intent
+
+object SessionSharer {
+    // String formattedSessions can be one or multiple sessions
+    @JvmStatic
+    fun shareSimple(context: Context, formattedSessions: String) {
+        val intent = Intent(Intent.ACTION_SEND).apply {
+            type = "text/plain"
+            putExtra(Intent.EXTRA_TEXT, formattedSessions)
+        }
+
+        // Show system's share UI. It handles the case of no matching apps.
```

Comment:
What happens exactly when no app is found?

Reply:
Depends on the Android version. Various ways of displaying a "No suitable apps found" message. On Android 11 it's this:

<img src="https://user-images.githubusercontent.com/218061/92171197-a4960e00-ee3b-11ea-87f5-6478077f9466.png" alt="EventFahrplan_share_fail" width=300>


You can easily force this by using a made up intent action value.

Reply:
OK. Thanks for the clarification.